README.md

```diff
@@ -36,3 +29,10 @@ match your theme perfectly.
 $ npm install
 $ grunt
 ```
+
+### Run tests
+```shell
+$ npm install
+$ bower install
```

Comment:
add this line to the `package.json` like here: https://github.com/gae-init/gae-init/blob/master/package.json#L10